### PR TITLE
Fix ui home screen

### PR DESCRIPTION
**Pull request by at-phatnguyen**
- add menu bar button

### CODE REVIEW

File: MyApp/View/Controllers/Home/HomeViewController.swift [hunk 1]
```diff
@@ -24,13 +24,11 @@ class HomeViewController: UIViewController {
...
 
     func configNaviBar() {
-
-        let navBar: UINavigationBar = UINavigationBar(frame: CGRect(x: 0, y: 0, width: 375, height: 64))
-        view.addSubview(navBar)
-        let profile = UIBarButtonItem(title: App.Strings.profile, style: .plain, target: self, action: #selector(didSelectProfileItem))
-        navigationItem.leftBarButtonItem = profile
-        let search = UIBarButtonItem(barButtonSystemItem: .search, target: self, action: #selector(didSelectSearchBarItem))
-        navigationItem.rightBarButtonItem = search
+        self.title = "HOME"
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: Please remove `self` + `define` string

File: MyApp/View/Controllers/Home/HomeViewController.swift [hunk 2]
```diff
@@ -24,13 +24,11 @@ class HomeViewController: UIViewController {
     }
 
     func configNaviBar() {
-
-        let navBar: UINavigationBar = UINavigationBar(frame: CGRect(x: 0, y: 0, width: 375, height: 64))
```
Comment (at-daiho) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: WARNING - Prefer object literals over image and color inits.

Comment (at-daiho) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: WARNING - Prefer object literals over image and color inits.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
Total linter violations in pull request: 3
Serious: 0
Warnings: 3

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request merged**